fastchat/model/model_adapter.py

```diff
@@ -1114,7 +1114,7 @@ class NousHermesAdapter(BaseModelAdapter):
     use_fast_tokenizer = False
 
     def match(self, model_path: str):
-        return "Nous-Hermes" in model_path
+        return "Nous-Hermes" in model_path.lower()
```

Comment:
Please double-check all your modifications. At least this one, Robin and RWKV are wrong. Convert all of them to lower cases.

Reply:
Done